engines/sci/engine/script_patches.cpp

```diff
@@ -2367,17 +2369,17 @@ static const uint16 kq5PatchWitchCageInit[] = {
 	PATCH_END
 };
 
-// The multilingual releases of KQ5 hang right at the end during the magic battle with Mordack.
-// It seems additional code was added to wait for signals, but the signals are never set and thus
-// the game hangs. We disable that code, so that the battle works again.
-// This also happened in the original interpreter.
-// We must not change similar code, that happens before.
-
+// The multilingual releases of KQ5 hang right at the end during the magic
+// battle with Mordack. It seems additional code was added to wait for signals,
+// but the signals are never set and thus the game hangs. We disable that code,
+// so that the battle works again. This also happened in the original
+// interpreter. We must not change similar code, that happens before.
+//
 // Applies to at least: French PC floppy, German PC floppy, Spanish PC floppy
 // Responsible method: stingScript::changeState, dragonScript::changeState, snakeScript::changeState
 static const uint16 kq5SignatureMultilingualEndingGlitch[] = {
 	SIG_MAGICDWORD,
-	0x89, 0x57,                      // lsg global[57h]
+	0x89, 0x57,                      // lsg global[87]
```

Comment:
I also don't really see a reason to change lines like these.
The original was consistent. Hexadecimal for both. And the ScummVM SCI-debugger accepts hex as well as decimal values for viewing globals.
Now it's mixed up, which makes it easier that errors (actual value mismatches) go unnoticed.